docs/topics/http/urls.txt

```diff
@@ -41,8 +41,8 @@ algorithm the system follows to determine which Python code to execute:
    :setting:`ROOT_URLCONF` setting.
 
 #. Django loads that Python module and looks for the variable
-   ``urlpatterns``. This should be a Python list of :func:`django.urls.path`
-   and/or :func:`django.urls.re_path` instances.
+   ``urlpatterns``. This should be an iterable (e.g. a list or tuple) of
```

Comment:
Rather than describing an iterable, WDYT about linking the Python docs:

```suggestion
   ``urlpatterns``. This should be an :term:`iterable` of
```

It seems to me we shouldn't need to re-explain what the Python docs already say.

Reply:
ok. to change.

Reply:
changed.